src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -170,8 +699,8 @@
  *
  * <h3>Notes</h3>
  * <ul>
- *     <li>This is a beta tool, and as such may generate errors or warnings.</li>
- *     <li>This tool is the successor to <a href="http://portals.broadinstitute.org/oncotator/">Oncotator</a>, with better support for germline data.</li>
+ *     <li>This tool is the spiritual successor to <a href="http://portals.broadinstitute.org/oncotator/">Oncotator</a>, with better support for germline data, numerous fixes for correctness, and many other features.</li>
+ *     <li>REMEMBER: <strong>Funcotator is NOT Oncotator.</strong></li>
```

Comment:
Can we get rid of this REMEMBER line?  I do not really think that needs clarification.

Reply:
I think it's somewhat important to temper expectations for a user that sees Funcotator and assumes it does the exact same thing that Oncotator does.

Reply:
I would not worry.  "spiritual successor" encompasses that.  Or verbiage more like "the feature set in Oncotator is not the same as Funcotator.", which is more specific.